erpnext/crm/doctype/crm_settings/crm_settings.json

```diff
@@ -14,7 +15,11 @@
   "column_break_2",
   "mark_opportunity_lost_after_days",
   "opportunity_auto_lost_reason",
-  "auto_mark_opportunity_as_lost"
+  "auto_mark_opportunity_as_lost",
+  "column_break_q3twi",
+  "maintenance_reminder_days_before_schedule",
+  "maintenance_reminder_time",
+  "auto_reminder_for_maintenance_schedule"
```

Comment:
Lets remove this checkbox for now, since this is already being checked with SMS Template